include/swift/Driver/Action.h

```diff
@@ -397,7 +397,7 @@ class StaticLinkJobAction : public JobAction {
 };
 
 class VerifyModuleInterfaceJobAction : public JobAction {
-  virtual void anchor();
+  void anchor() override;
```

Comment:
Note this is the only fix, it was missing override which triggered `-Wsuggest-override`. The rest of the changes were for convention/consistency.